frontend/forms/OBSBasic.ui

```diff
@@ -199,7 +199,7 @@
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="class" stdset="0">
-              <string>label-preview-title</string>
+              <string>previewProgramLabels</string>
```

Comment:
@Warchamp7 might have an opinion on this, but I think CSS/QSS class names should be singular (not plural). Also, if changing this, wouldn't the class rule in the theme files need to be changed? Or was this changed by mistake?

Reply:
I think that's an automatic change through rebasing though, the only classes that needed to be changed are the actual C++ class names in the `customwidgets` tree.

Reply:
I'm not sure I follow.

There's a distinction here between the **C++ class** the widget is tied to, and the _**dynamic property**_ **class** on the widget, which is used for styling purposes.

The thing changed here is a dynamic property that seems like an accidental modification.

[Existing line on master](https://github.com/obsproject/obs-studio/blob/f03280c4fd449d1591363291fd2762ffca878e6a/UI/forms/OBSBasic.ui#L202)
[The styling rule in our obt file](https://github.com/obsproject/obs-studio/blob/f03280c4fd449d1591363291fd2762ffca878e6a/UI/data/themes/Yami.obt#L1515)

Reply:
That's what I'm saying, yes. The line should not have changed and the only reason it did was because git was able to apply the change when I rebased it to master.